### PR TITLE
added nepali translation of 5-data-science-in-cloud/19-azure

### PR DESCRIPTION
**Pull request by swapnilregmi**
#174

### CODE REVIEW

**Comment by swapnilregmi** [comment]
@paladique please verify my translation

File: 5-Data-Science-In-Cloud/19-Azure/translations/assignment.ne.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Azure ML SDK प्रयोग गरेर डाटा विज्ञान परियोजना
+
+## निर्देशन
+
+हामीले Azure ML SDK सँग मोडेललाई तालिम, डिप्लोय, र उपभोग गर्न Azure ML प्लेटफर्म कसरी प्रयोग गर्ने भनेर हेर्यौं। अब केहि डेटा को लागी वरिपरि हेर्नुहोस् जुन तपाईले अर्को मोडेललाई प्रशिक्षित गर्न, प्रयोग गर्न र उपभोग गर्न सक्नुहुन्छ। तपाईंले Kaggle र [Azure Open Datasets](https://azure.microsoft.com/services/open-datasets/catalog?WT.mc_id=academic-40229. -cxa&ocid=AID3041109) मा डाटासेट फेला पार्न सक्नुहुन्छ।
```
Comment (paladique) [inline 5-Data-Science-In-Cloud/19-Azure/translations/assignment.ne.md:5]: the link for Azure open datasets has a space, which may break the link. It should be `https://azure.microsoft.com/services/open-datasets/catalog?WT.mc_id=academic-40229-cxa&ocid=AID3041109`

**Review by paladique — CHANGES_REQUESTED** [review]

File: 5-Data-Science-In-Cloud/19-Azure/translations/assignment.ne.md (hunk 1 above)
Comment (swapnilregmi) [reply 5-Data-Science-In-Cloud/19-Azure/translations/assignment.ne.md]: I have updated my file. Please verify.

**Review by paladique — APPROVED** [review]

**Comment by paladique** [comment]
@swapnilregmi thank you!

**Pull request merged**